pygmt/src/meca.py

```diff
@@ -231,8 +231,9 @@ def meca(
         determined by the z-value (i.e., event depth or the third column for
         an input file).
     no_clip : bool
-        Does NOT skip symbols that fall outside frame boundary specified by
-        ``region`` [Default is False, i.e. plot symbols inside map frame only].
+        Do NOT skip symbols that fall outside frame boundary specified by
```

Comment:
```suggestion
        Do NOT skip symbols that fall outside the frame boundary specified by
```

Reply:
@michaelgrund in commit https://github.com/GenericMappingTools/pygmt/pull/2376/commits/2f94dd5d377d1731b87d3a5830d97fb6742a06b3 I _tried_ to make the docstrings for `no_clip` more consistent across the single methods.